### DISCUSSION TITLE
Eigene Templates im Theme-Builder duplizieren

### DISCUSSION THREAD

**Discussion by Amulashtra** [elementor/elementor#22009] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Wenn ich ein Template für z.B. eine Kategorie im Theme-Builder erstelle und das gleiche Template mit nur wenigen Änderungen für eine andere Kategorie duplizieren möchte, geht das nur über den Umweg Export/Import.

### Describe the solution you'd like

Warum kann man im Theme-Builder seine eigenen Templates nicht mit einem Klick duplizieren?

### Describe alternatives you've considered

Ich habe leider auch noch kein Plugin gefunden, das Elementor-Templates im Theme-Builder duplizieren kann.

### Additional context

![7A5BEA34-A8DB-4F64-BD96-D2C81EB813B1](https://user-images.githubusercontent.com/130484671/231217051-de55dcf3-a84e-4dad-8781-db94dab68337.jpeg)

**Comment by pumadog** [comment]
Weiß nicht ob deutsch hier gern gesehen wird…
I usually make a new template and import (file symbol in the section) the existing one into it. It's a copy then.

**Comment by Amulashtra** [comment]
ok, i'll try the google translator.
I'm sure you mean a template. but I mean the theme builder. if they had an easy way to duplicate their templates there, that would be great.

**Reply by pumadog** [reply]
In the theme builder, you can insert an existing template into a section. For example you add a new header. When you insert an existing header template into it (file symbol in the section/container) the settings get imported as well and you have a 2nd theme builder header.
